docs/low-code-sql/development/gems/joins.md

```diff
@@ -10,6 +10,36 @@ tags:
   - transformation
 ---
 
-:::caution
-This detailed page all about SQL Joins is under construction. Please pardon our dust.
+Upon opening the join Gem, you can see a pop-up which provides several helpful features.
+
+![Join definition](img/JoinCondition.png)
+
+For transparency, you can always see the **(1) Input schema** on the left hand-side, **(2) Errors** in the footer, and have the ability to **(3) Run** the Gem on the top right.
+
+To fill-in our **(5) Join condition** within the **(4) Conditions** section, start typing the input table name and key. For example, if we have two input tables, `nation` and `customer`, type `nation.nationkey = customers.nationkey`. This condition finds a nation based on the nationkey feild for every single customer.
+
+When you’re writing your join conditions, you’ll see available functions and columns to speed up your development. When the autocomplete appears, press ↑, ↓ to navigate between the suggestions and press tab to accept the suggestion.
+
+Select the **(6)Join Type** according to the provider, eg [Databricks](https://docs.databricks.com/en/sql/language-manual/sql-ref-syntax-qry-select-join.html) or [Snowflake.](https://docs.snowflake.com/en/user-guide/querying-joins)
+
```

Comment:
@several27 you had suggested we account for the differences between the databricks and snowflake join types and implementation, but I'm not clear on these differences. Is it ok if we push this version now, and as we get complicated join questions we can update this page?